src/styles/1c-light.css

```diff
@@ -0,0 +1,133 @@
+/*!
+  Theme: 1c-light
+  Description: Style IDE 1C:Enterprise 8
+  Author: (c) Barilko Vitaliy <barilkovetal@gmail.com>
+  Maintainer: @Diversus23
+  Website: https://softonit.ru/
+  License: see project LICENSE
+  Touched: 2023
+*/
+
+pre code.hljs {
+    display: block;
+    overflow-x: auto;
+    padding: 1em;
+  }
+  
+  code.hljs {
+    padding: 3px 5px;
+  }
+  /* end baseline CSS */
+  
+  .hljs {
+    font-family: "Courier New", monospace; /* This font is used in IDE the 1C:Enterprise 8 */
+    color: #0000ff;
+    background: #ffffff;
+    font-weight: normal;
+    font-style: normal;
+  }
```

Comment:
There is no need for these `normal` resets and our themes are not allowed to change the font.